<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleTraceDialog</class>
 <widget class="QDialog" name="SimpleTraceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="lblLogText">
     <property name="text">
      <string>&amp;Log Text:</string>
     </property>
     <property name="buddy">
      <cstring>editLogText</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="HistoryLineEdit" name="editBreakCondition"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblLogCondition">
     <property name="text">
      <string>Lo&amp;g Condition:</string>
     </property>
     <property name="buddy">
      <cstring>editLogCondition</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="HistoryLineEdit" name="editSwitchCondition">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblSwitchCondition">
     <property name="text">
      <string>Switch Condition:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnLogFile">
       <property name="text">
        <string>Log &amp;File...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>C&amp;ancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblMaxTraceCount">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Maximum trace count:</string>
     </property>
     <property name="buddy">
      <cstring>spinMaxTraceCount</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="HistoryLineEdit" name="editLogCondition">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="HistoryLineEdit" name="editCommandCondition"/>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="lblHint">
     <property name="text">
      <string>Hint: History is available in every text field with the Up/Down arrows!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="spinMaxTraceCount">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="HistoryLineEdit" name="editLogText"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblCommandText">
     <property name="text">
      <string>&amp;Command Text:</string>
     </property>
     <property name="buddy">
      <cstring>editCommandText</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblBreakCondition">
     <property name="text">
      <string>&amp;Break Condition:</string>
     </property>
     <property name="buddy">
      <cstring>editBreakCondition</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="HistoryLineEdit" name="editCommandText"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblCommandCondition">
     <property name="text">
      <string>C&amp;ommand Condition:</string>
     </property>
     <property name="buddy">
      <cstring>editCommandCondition</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Trace Over System Calls</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistoryLineEdit</class>
   <extends>QLineEdit</extends>
   <header>HistoryLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>editBreakCondition</tabstop>
  <tabstop>editLogText</tabstop>
  <tabstop>editLogCondition</tabstop>
  <tabstop>editCommandText</tabstop>
  <tabstop>editCommandCondition</tabstop>
  <tabstop>spinMaxTraceCount</tabstop>
  <tabstop>editSwitchCondition</tabstop>
  <tabstop>btnLogFile</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>SimpleTraceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
